### PR TITLE
E-commerce project

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/account.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RK-Accessories</title>
+    <link rel="stylesheet" href="account.css" />
+  </head>
+  <body>
+    <h2>
+      /* The Page you are trying to acccess is currently under construction! */
+    </h2>
+    <div class="maintenance-img">
+      <img
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/account.html:15]: Alt?

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  text-align: center;
+  background: radial-gradient(#fff, rgb(53, 184, 53));
+  height: 250px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 40px;
+  margin-top: 40px;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.css:18]: Group Similar Properties @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  </head>
+  <body>
+    <div class="header">
+      <h1>Get In Touch With Us</h1>
+
+      <p>Want to conncet with us? Here is how you can reach us.</p>
+    </div>
+    <div class="row">
+      <div class="col-3-1">
+        <img src="ferguson images/email-us-1805514_960_720.webp" />
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html:18]: Add Alt attribute @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <h5><a href="">rk116159@gmail.com</a></h5>
+      </div>
+      <div class="col-3-1">
+        <img src="ferguson images/call-us-1817506_960_720.webp" />
+        <h5><a href="">+91 8655231127</a></h5>
+      </div>
+      <div class="col-3-3">
+        <img src="ferguson images/linked-in-2668692_960_720.webp" />
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html:26]: Alt

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        <h5>
+          <a
+            href="https://www.linkedin.com/in/rehan-khan-engineer/"
+            target="_blank"
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html:30]: What will this attribute do ? @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+      href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="header">
+      <div class="container">
+        <div class="navbar">
+          <div class="logo">
+            <img src="Images/logo.png" width="125px" />
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html:24]: Remove inline styling and also add Alt attribute @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+          <nav>
+            <ul id="MenuItems">
+              <li><a href="index.html">Home</a></li>
+              <li><a href="products.html" target="_blank">Products</a></li>
+              <li><a href="account.html" target="_blank">About</a></li>
+              <li><a href="contact.html" target="_blank">Contact</a></li>
+              <li><a href="account.html" target="_blank">Account</a></li>
+            </ul>
+          </nav>
+          <img src="images/cart.png" width="30px" height="30px" />
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html:35]: Remove inline styling
@Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/products.html [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+
+    <div class="small-container">
+      <div class="row row-2">
+        <h2>All Products</h2>
+        <select>
+          <option>Default Shorting</option>
+          <option>Short By Popularity</option>
+          <option>Short By Rating</option>
+          <option>Short By Sales</option>
+          <option>Short By Price</option>
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/products.html:48]: Sort by ?

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Remove inline stylings and add Alt attribute to images also correct some words

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/style.css [hunk 9]
```diff
@@ -0,0 +1,384 @@
...
+  content: "";
+  background: #ff523b;
+  width: 80px;
+  height: 5px;
+  border-radius: 5px;
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+}
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/style.css:142]: Group Similar Properties @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/account.html (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/account.html:15]: Done

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.css (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.css]: done

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html (hunk 4 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html]: done

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html (hunk 6 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html]: done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/products.html (hunk 8 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/products.html]: done

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/style.css (hunk 9 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/style.css]: done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html (hunk 3 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html]: done

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html (hunk 5 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html:33]: _self
 @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html (hunk 7 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/index.html]: Tried but its affecting responsive behavior @Abbas4tech

**Review by Abbas4tech — DISMISSED** [review]
Please add Alt to images of every html document and group Similar Properties of every selector

**Comment by Rehan-Khan-Engineer** [comment]
> Please add Alt to images of every html document and group Similar Properties of every selector

Done

**Review by mujeeb100 — APPROVED** [review]

**Review by Naufil-Shemle — APPROVED** [review]
Nice one

**Review by shaikh-7abish — APPROVED** [review]

**Pull request merged**